db/tables.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS user(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    username VARCHAR(255),
+    password VARCHAR(255),
+    email VARCHAR(255),
+    full_name VARCHAR(255),
+    address VARCHAR(255),
+    phone VARCHAR(255)
+);
+
+INSERT IGNORE INTO user (id, username, password, email, full_name, address, phone) VALUES (1, "admin", "admin", "admin", "admin", "admin", "admin");
+
```

Comment:
If you feel able to improve the length of each field, maybe it would be interesting to define it already, it is not necessary that you do it, just a note... think that you will come many more projects, and you have a deadline to deliver.

Reply:
Noted, I have improved the length of each field.